.circleci/config.yml

```diff
@@ -11,11 +12,25 @@ jobs:
       - node/install-packages
```

Comment:
this line can be replaced with `ship/node-install-packages` which relays the call to the circleci/node orb.